packages/headless/src/controllers/commerce/facets/core/headless-core-commerce-facet.test.ts

```diff
@@ -1,6 +1,7 @@
 import {commerceFacetSetReducer as commerceFacetSet} from '../../../../features/commerce/facets/facet-set/facet-set-slice';
```

Comment:
@Spuffynism I didn't really change anything in this test suite (yet).

I'm having a look at it this morning (Monday, Dec. 4) to see if the test coverage is still OK given the refactoring I've done.